### PR TITLE
Minor documentation changes

### PR DESCRIPTION
**Pull request by kerneltime**
## Description
Minor documentation cleanup.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed

### CODE REVIEW

**Comment by harshavardhana** [comment]
@kerneltime as practice avoid creating branches on origin repo, it is better to have branches in fork to be submitted as PRs

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: docs/bucket/replication/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Bucket Replication Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
 
-Bucket replication is designed to replicate specific objects in buckets that are configured for replication.
+Bucket replication is designed to replicate selected objects in a bucket to a destination bucket.
 
-To replicate objects in a bucket to a destination bucket on a target site either on the same cluster or a different cluster, start by creating version enabled buckets on both `source` and `dest` buckets. Next, the target site and destination bucket need to be configured on MinIO server by setting
+To replicate objects in a bucket to a destination bucket on a target site either in the same cluster or a different cluster, start by enabling [versioning](https://github.com/minio/minio/tree/master/docs/bucket/versioning) for both source and destination buckets. Finally, the target site and the destination bucket need to be configured on the source MinIO server.
```
Comment (harshavardhana) [inline docs/bucket/replication/README.md]: this link to docs is perhaps should be https://docs.minio.io/docs/minio-bucket-versioning-guide.html

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :x: [more...](#60b6fb18-1d8a-4471-9412-6a6f4472695f) |
<a name="60b6fb18-1d8a-4471-9412-6a6f4472695f"></a>
### 10209-ba0b9d0/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev4.minio.io:30015
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp f056e9242eff:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 1 seconds
(5/15) Running awscli tests ... done in 1 minutes and 4 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 24 seconds
(8/15) Running minio-dotnet tests ... done in 29 seconds
(9/15) Running minio-go tests ... done in 45 seconds
(10/15) Running minio-java tests ... FAILED in 17 seconds
{
  "name": "minio-java",
  "function": "listIncompleteUploads(ListIncompleteUploadsArgs args)",
  "duration": 303,
  "status": "FAIL",
  "error": "java.net.SocketException: Connection reset by peer (Write failed) >>> [java.base/java.net.SocketOutputStream.socketWrite0(Native Method), java.base/java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:110), java.base/java.net.SocketOutputStream.write(SocketOutputStream.java:150), okio.Okio$1.write(Okio.java:79), okio.AsyncTimeout$1.write(AsyncTimeout.java:180), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:179), okio.RealBufferedSink.write(RealBufferedSink.java:42), okhttp3.internal.http1.Http1ExchangeCodec$KnownLengthSink.write(Http1ExchangeCodec.java:324), okio.ForwardingSink.write(ForwardingSink.java:35), okhttp3.internal.connection.Exchange$RequestBodySink.write(Exchange.java:231), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:179), okio.RealBufferedSink.write(RealBufferedSink.java:117), io.minio.HttpRequestBody.writeTo(HttpRequestBody.java:78), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:69), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:43), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117), okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229), okhttp3.RealCall.execute(RealCall.java:81), io.minio.MinioClient.execute(MinioClient.java:1014), io.minio.MinioClient.uploadPart(MinioClient.java:7781), io.minio.MinioClient.putObject(MinioClient.java:4706), io.minio.MinioClient.putObject(MinioClient.java:4881), FunctionalTest.listIncompleteUploads_test1(FunctionalTest.java:1555), FunctionalTest.runTests(FunctionalTest.java:3814), FunctionalTest.main(FunctionalTest.java:4056)]"
}
(10/15) Running minio-js tests ... done in 45 seconds
(11/15) Running minio-py tests ... done in 1 minutes and 8 seconds
(12/15) Running s3cmd tests ... done in 16 seconds
(13/15) Running s3select tests ... done in 2 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Comment by poornas** [comment]
can you rebase this PR? ReplicationArn was renamed to Role which is in conformance with S3 spec

**Review by kannappanr — APPROVED** [review]

**Comment by kerneltime** [comment]
> can you rebase this PR? ReplicationArn was renamed to Role which is in conformance with S3 spec

Done

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by poornas — APPROVED** [review]

**Pull request merged**